lib/commands/source.js

```diff
@@ -13,16 +13,14 @@ const commands = {
       return await this.executeAtom('execute_script', [script, []]);
     }
 
-    if ((await this.settings.getSettings()).useJSONSource) {
-      const srcTree = await this.mobileGetSource('json');
+    const { pageSourceExcludedAttributes: excludedAttributes, useJSONSource: useJSONSource } = await this.settings.getSettings();
```

Comment:
`useJSONSource: useJSONSource` is redundant. This syntax is only used if you need to rename deconstructed variable. If the name remains the same then the following syntax should be used:

```js
const { 
  pageSourceExcludedAttributes: excludedAttributes,
  useJSONSource,
} = await this.settings.getSettings();
```

(please keep formatting as well)